### ISSUE TITLE
Hindi characters are shown as boxes

### ISSUE THREAD

**Issue by kavitakanojiya** [mileszs/wicked_pdf#496]
Any idea why `हिन्दी ` is shown as boxes:

![screen shot 2016-03-07 at 7 34 01 pm](https://cloud.githubusercontent.com/assets/1167590/13571524/877ba874-e49b-11e5-9664-d04b8754e176.png)

**Comment by unixmonkey** [comment]
It is likely because the server doesn't have those fonts installed.

See this issue for a similar issue/solution description:
https://github.com/mileszs/wicked_pdf/issues/456

**Comment by aldrienht** [comment]
Yes, @kavitakanojiya,  i think you need to manually install the font on Server, I encountered the same problem when using Japanese fonts.

**Comment by jd1386** [comment]
Almost same thing happened to Korean language. sudo apt-get install fonts-nanum fixed it.

**Comment by Ali-raza-khan** [comment]
I am having this problem. Thai fonts are not loading amd showed me boxes.
![Screen Shot 2019-07-31 at 3 59 33 PM](https://user-images.githubusercontent.com/10446457/62206995-6a427d80-b3ac-11e9-9fa6-30cffc7caacc.png)

**Comment by unixmonkey** [comment]
@Ali-raza-khan Do you have them installed on your PDF-generating servers?

A quick web search turns up `sudo apt-get install fonts-thai-tlwg` to install Thai fonts, if you are using an apt-based system.

**Comment by vijay-gurjar** [comment]
> Any idea why `हिन्दी` is shown as boxes:
> 
> ![screen shot 2016-03-07 at 7 34 01 pm](https://cloud.githubusercontent.com/assets/1167590/13571524/877ba874-e49b-11e5-9664-d04b8754e176.png)


for Hindi fonts
sudo apt-get install fonts-indic

**Comment by neerajkumarjarvis** [comment]
sudo apt-get install fonts-indic

this is working 👍

**Comment by neerajkumarjarvis** [comment]
> > Any idea why `हिन्दी` is shown as boxes:
> > ![screen shot 2016-03-07 at 7 34 01 pm](https://cloud.githubusercontent.com/assets/1167590/13571524/877ba874-e49b-11e5-9664-d04b8754e176.png)
> 
> for Hindi fonts sudo apt-get install fonts-indic

this is working 👍
